### ISSUE TITLE
🐞 - `Tabs` with "more" section fails to close dropdown on keyboard navigation without `provideAnimations()`

### ISSUE THREAD

**Issue by nsbarsukov** [taiga-family/taiga-ui#12728]
### Reproduction url

https://taiga-ui.dev/next/navigation/tabs#complex

### Description

1. Remove `provideAnimations()` from application config
https://github.com/taiga-family/taiga-ui/blob/a1b4622180e066a598346b25d78e89297b0d1a34/projects/demo/src/modules/app/app.config.ts#L67-L69
2. Open locally this example
https://taiga-ui.dev/next/navigation/tabs#complex
3. Click on ellipsis icon (more section)
4. Press ArrowDown
5. Press Enter

Dropdown is still opened

https://github.com/user-attachments/assets/b8049a16-5e9b-42e9-86f5-e2c0bdc2f220

### Taiga UI version

main branch

### Browser Used

Chrome

### OS Used

macOS

**Comment by nsbarsukov** [comment]
Possibly relates:
* https://github.com/taiga-family/taiga-ui/issues/9836

**Comment by nsbarsukov** [comment]
Fixed by:
* https://github.com/taiga-family/taiga-ui/pull/12860
